src/myrtlespeech/data/sampler.py

```diff
@@ -1,14 +1,37 @@
 import random
 
 
-class RandomBatchSampler:
-    """TODO"""
+class SequentialRandomSampler:
+    """A sequential or random iterable over batches.
+    The iterator used each time this iterable is iterated over will yield
```

Comment:
Newline above this.  When we were working on `deepspeech_internal` sam sent me the following link and asked me to re-write all docstrings in this format: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Fixed it! Thanks for the link